embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/comms/delivery/EmbraceDeliveryService.kt

```diff
@@ -190,7 +192,8 @@
         sessionMessage.session.endTime
             ?: sessionMessage.session.terminationTime
             ?: sessionMessage.session.lastHeartbeatTime
-            ?: sessionMessage.session.startTime
+            ?: sessionMessage.getSessionSpan()?.startTimeNanos?.nanosToMillis()
+            ?: -1
```

Comment:
We should get the backend to have a metric to detect an endTime of -1 so this basically impossible condition is caught